src/test/java/org/roaringbitmap/buffer/TestImmutableRoaringBitmap.java

```diff
@@ -82,18 +96,19 @@ public void testContains() throws IOException {
     }
     
     @SuppressWarnings("resource")
-	static ByteBuffer serializeRoaring(MutableRoaringBitmap mrb) throws IOException {
-			ByteBuffer outbb = ByteBuffer.allocate(mrb.serializedSizeInBytes());
+	static ByteBuffer serializeRoaring(ImmutableRoaringBitmap mrb) throws IOException {
+            byte[] backingArray = new byte[mrb.serializedSizeInBytes()+1024];
+			ByteBuffer outbb = ByteBuffer.wrap(backingArray, 1024, mrb.serializedSizeInBytes()).slice();
 			DataOutputStream dos = new DataOutputStream(new OutputStream(){
 	        ByteBuffer mBB;
 	        OutputStream init(ByteBuffer mbb) {mBB=mbb; return this;}
 	            public void write(int b) {mBB.put((byte) b);}
-	            public void write(byte[] b) {}            
-	            public void write(byte[] b, int off, int l) {}
+	            public void write(byte[] b) {}
+	            public void write(byte[] b, int off, int l) { mBB.put(b,off,l); }
 	        }.init(outbb));
 			mrb.serialize(dos);
 			dos.close();
-			
+
```

Comment:
I'd like to cleanup some of the indentation in this method but wasn't sure if I should do that at the same time as functionality changes.